### ISSUE TITLE
Report brms refitting models

### ISSUE THREAD

**Issue by jgeller112** [easystats/report#374]
Maybe I am missing something, but is there a way to avoid always refitting brms models using the report function? I have pretty large models and using report will either error out or run forever.

**Comment by strengejacke** [comment]
@rempsyc @mattansb @DominiqueMakowski I'm not sure, but I guess this is due to computing Bayes factors when retrieving the performance-table? We should try to find the bottle neck here, maybe using the _profile_ package.

@jgeller112 do you have a model you can share, so we can try to reproduce your issue? Else, maybe you find a model out of our pre-compiled models from here: https://easystats.github.io/circus/reference/index.html and just name one of those example models that behaves similarly.

**Comment by jgeller112** [comment]
Here you go: 
```
fit_c <- read_rds("https://osf.io/hmqpc/download")
```

```
report(fit_c)
```

**Comment by strengejacke** [comment]
If I understand the profiling right, one issue is `logLik()`:

![image](https://github.com/easystats/report/assets/26301769/af0c0507-dd44-4289-9a4b-663c34f8f823)

And another bottle neck is `rstan::summary()`:

![image](https://github.com/easystats/report/assets/26301769/ad23f665-fcaf-462a-aedc-4557def30b00)

Given that already the simple call to `summary()` takes ~ 7 seconds on my machine, this looks like the issue is due to the very complex model. The profile-file is attached (I used _profvis_), maybe we find ways to remove some of those time-consuming things.

[report_brms.zip](https://github.com/easystats/report/files/11890810/report_brms.zip)

**Comment by DominiqueMakowski** [comment]
> I think this is because report standardizes coefficients using the refit method (which it 100% should not do since it does not adjust the priors accordingly).

yes, the standardization method should be changeable to something else than "refit" (?)

**Comment by strengejacke** [comment]
@jgeller112 Can you then try to run `report(fit_c, include_effectsize = FALSE)`?

**Comment by jgeller112** [comment]
I get 
```
Error in report_table.brmsfit(x, include_effectsize = FALSE, ...) : 
  formal argument "include_effectsize" matched by multiple actual arguments
```

**Comment by ZoeGold** [comment]
Has there been a fix for this issue in the meantime? I am running into the exact same issue with my brms models (and not just one model, but a wide range of them)

**Comment by utkuturk** [comment]
I am having the same problem with brms. Even though model fit with a file specified, it refits all of the models I am using, even the simplest intercept only models. Is there a way to prohibit this behavior? Or something that I could do while fitting the models, so it does not refits it?

**Comment by rempsyc** [comment]
Oops, I was still on the old version. On version 0.6.1.5, there _is_ a major improvement: an almost 4x reduction from ~4h to ~50 min:

```r
library(bench)
library(tidyverse)
library(easystats)
library(rstan)
packageVersion("report")
#> [1] '0.6.1.5'

mark(
  fit_c <- read_rds("https://osf.io/hmqpc/download"),
  report_model(fit_c, effectsize_method = "refit"),
  report_model(fit_c),
  max_iterations = 1,
  check = FALSE
)
#> The desired updates require recompiling the model
#> Start sampling
#> Running MCMC with 4 sequential chains, with 4 thread(s) per chain...
#> 
#> Chain 1 Iteration:    1 / 5000 [  0%]  (Warmup) 
[...]
#> Chain 4 Iteration: 5000 / 5000 [100%]  (Sampling) 
#> Chain 4 finished in 2612.8 seconds.
#> The desired updates require recompiling the model
#> Start sampling
#> Running MCMC with 4 sequential chains, with 4 thread(s) per chain...
#> 
#> Chain 1 Iteration:    1 / 5000 [  0%]  (Warmup) 
[...]
#> Chain 4 Iteration: 5000 / 5000 [100%]  (Sampling) 
#> Chain 4 finished in 2671.7 seconds.
#> 
#> # A tibble: 3 × 6
#>   expression                             min median `itr/sec` mem_alloc `gc/sec`
#>   <bch:expr>                          <bch:> <bch:>     <dbl> <bch:byt>    <dbl>
#> 1 "fit_c <- read_rds(\"https://osf.i… 22.35s 22.35s 0.0447     914.81MB  0      
#> 2 "report_model(fit_c, effectsize_me…  4.18h  4.18h 0.0000664     1.2TB  0.00571
#> 3 "report_model(fit_c)"               49.73m 49.73m 0.000335     1.25TB  0.0255
Created on 2025-09-22 with [reprex v2.1.1](https://reprex.tidyverse.org/)
```
